### PR TITLE
Wizard Dialog -> default percentage increment is 5% rather than 1%

### PR DESCRIPTION
**Pull request by erobinson**
…. the 1 percent changes were too slow for me.

### CODE REVIEW

**Review by Andries-Smit — CHANGES_REQUESTED** [review]
Please review the comments.
After this change is merged, I make sure that the settings is also synced with AAPS watch, 
Similar to #1195

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1139,5 +1139,8 @@
     <string name="identification">Identification (email, FB or Discord nick etc)</string>
     <string name="identification_not_set">Identification not set in dev mode</string>
     <string name="not_available_full">Not available</string>
+    <string name="key_boluswizard_percentage_increment_size">1</string>
+    <string name="partialboluswizardincrementsize">Change the correction percentage by this much</string>
```
[the remark this answers is not in the document]
Comment (erobinson) [reply app/src/main/res/values/strings.xml]: corrected

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1139,5 +1139,8 @@
     <string name="identification">Identification (email, FB or Discord nick etc)</string>
     <string name="identification_not_set">Identification not set in dev mode</string>
     <string name="not_available_full">Not available</string>
+    <string name="key_boluswizard_percentage_increment_size">1</string>
```
[the remark this answers is not in the document]
Comment (erobinson) [reply app/src/main/res/values/strings.xml]: corrected

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1139,5 +1139,8 @@
...
+    <string name="partialboluswizardincrementsize">Change the correction percentage by this much</string>
+    <string name="deliverpartofboluswizardincrementsize"><![CDATA[Each time you click the buttons to change the correction percentage, it will go up or down by this much. So a value of 5 will go 100 -> 105 -> 110 -> 115 with each up click.]]></string>
```
[the remark this answers is not in the document]
Comment (erobinson) [reply app/src/main/res/values/strings.xml]: corrected

File: app/src/main/res/xml/pref_overview.xml [hunk 4]
```diff
@@ -28,6 +28,11 @@
                 android:key="@string/key_show_wizard_button"
                 android:title="@string/overview_calculator_label" />
 
+            <SwitchPreference
+                android:defaultValue="false"
+                android:key="@string/key_show_profile_button"
```
[the remark this answers is not in the document]
Comment (erobinson) [reply app/src/main/res/xml/pref_overview.xml]: It's not, it's something. else I was working on that I accidentally committed.

**Comment by Andries-Smit** [comment]
@erobinson we could introduce another setting, but do we have to? Should the default for all users be just 5% per click on both the phone and watch app?

**Comment by erobinson** [comment]
> @erobinson we could introduce another setting, but do we have to? Should the default for all users be just 5% per click on both the phone and watch app?

defaulting to 5% would definitely make it easier. I thought 1% had been done for a reason.

**Comment by erobinson** [comment]
@Andries-Smit @MilosKozak - I think this should be good to merge now.

**Comment by Andries-Smit** [comment]
Good to go!

**Pull request merged**